modules/client/src/controllers/BuyController.test.ts

```diff
@@ -28,7 +28,7 @@ describe('BuyController: unit tests', () => {
   //         amount: { amountToken: '1', amountWei: '0' },
   //         type: 'PT_CHANNEL',
   //         meta: {},
-  //         recipient: '$$HUB$$',
+  //         recipient: process.env.HUB_ADDRESS,
```

Comment:
@rhlsthrm so i know this was a hack to inject the hub address, changing this wouldnt cause any issues hub side right?